packages/lib/src/toolbar/controls/Selector/Selector.tsx

```diff
@@ -77,7 +77,7 @@ function Selector<T extends string>(props: Props<T>) {
         aria-labelledby={`${label ? labelId : ''} ${currentOptionId}`}
         aria-haspopup="listbox"
         aria-expanded={isOpen}
-        aria-controls={(isOpen && context.floatingId) || undefined}
```

Comment:
This comes from `unicorn/consistent-destructuring`, which detects property access on a variable that has already been partially destructured. Unfortunately, I had to later disable it because it would give a lot of false positives with type guards like `isDataset(entity) && entity.chunks`